packages/cryptoplease_link/cryptoplease_api/jupiter_aggregator/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: jupiter_aggregator
+description: A new Flutter package project.
+version: 0.0.1
+
+environment:
+  sdk: '>=2.18.0 <3.0.0'
+  flutter: ">=1.17.0"
```

Comment:
Why should it depend on flutter at all?

Reply:
Flutter package default dependencies. Removed!